MovingBlocks/TerasologyLauncher
# feat: add missing german translations

### Contains

- Additional label translations for German (de) locale
- Fix for a typo in an existing label translation

### How to test

1. Start the launcher and set the locale to German ("Deutsch", de)
2. Verify that the following labels are correctly translated:
    - settings view: logging level label
    - settings view: show pre-releases and nightly builds checkbox
    - settings view: advanced options details section title
    - main view: tab titles
